Packages/vcs/Lib/Canvas.py

```diff
@@ -2679,11 +2679,25 @@ def plot_filledcontinents(self,slab,template_name,g_type,g_name,bg,ratio):
         except Exception,err:
             print err
 
+    def __new_elts(self,original,new):
```

Comment:
@doutriaux1 if this is urgent, I would suggest you pass the branch name to the folks that need it (or create staging). I think it would be better if we take care of the code at the review level.